frontend/src/services/index.ts

```diff
@@ -13,9 +23,9 @@ const BASE_URL = import.meta.env.VITE_API_URL;
 export async function authorized_fetch<T>(
     endpoint: string,
     requestOptions: RequestInit,
-    omitContentType: boolean = false,
-    json: boolean = true
+    options: FetchOptions = {}
```

Comment:
`download_file` functie in `src/utils.ts` is nu kapot.

Reply:
fixed it